### PR TITLE
Bluetooth: audio: Fix error message in PACS

### PR DESCRIPTION
**Pull request by szymon-czapracki**
Change error code in PACS source/sink write operation to Write Request Rejected.

### CODE REVIEW

**Comment by szymon-czapracki** [comment]
@MariuszSkamra FYI

**Comment by Thalley** [comment]
@szymon-czapracki I don't think I agree with this change. What's the rationale for changing these? PTS tests failing? And if so, does the test spec specifically require this particular result code?

**Comment by szymon-czapracki** [comment]
> @szymon-czapracki I don't think I agree with this change. What's the rationale for changing these? PTS tests failing? And if so, does the test spec specifically require this particular result code?

Exactly as you suspected, PTS tests are failing. But there is also a description of proper behaviour in spec:
image

File: subsys/bluetooth/audio/pacs.c [hunk 1]
```diff
@@ -303,29 +303,29 @@ static ssize_t snk_loc_write(struct bt_conn *conn,
 	enum bt_audio_location location;
 
 	if (offset) {
-		return BT_GATT_ERR(BT_ATT_ERR_INVALID_OFFSET);
+		return BT_GATT_ERR(BT_ATT_ERR_WRITE_REQ_REJECTED);
```
Comment (Thalley) [inline subsys/bluetooth/audio/pacs.c:306]: Keep as `BT_ATT_ERR_INVALID_OFFSET` IMO

File: subsys/bluetooth/audio/pacs.c [hunk 2]
```diff
@@ -399,29 +399,29 @@ static ssize_t src_loc_write(struct bt_conn *conn,
 	uint32_t location;
 
 	if (offset) {
-		return BT_GATT_ERR(BT_ATT_ERR_INVALID_OFFSET);
+		return BT_GATT_ERR(BT_ATT_ERR_WRITE_REQ_REJECTED);
```
Comment (Thalley) [inline subsys/bluetooth/audio/pacs.c:402]: Keep as `BT_ATT_ERR_INVALID_OFFSET` IMO

**Review by Thalley — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/audio/pacs.c (hunk 2 above)
Comment (szymon-czapracki) [reply subsys/bluetooth/audio/pacs.c]: Done

**Review by Thalley — DISMISSED** [review]

**Review by MariuszSkamra — DISMISSED** [review]

**Review by Thalley — APPROVED** [review]

**Review by sjanc — APPROVED** [review]

**Pull request merged**